include/mbedtls/ssl.h

```diff
@@ -4360,6 +4360,24 @@ void mbedtls_ssl_conf_cert_req_ca_list(mbedtls_ssl_config *conf,
  *                 with \c mbedtls_ssl_read()), not handshake messages.
  *                 With DTLS, this affects both ApplicationData and handshake.
  *
+ * \note           Defragmentation of incoming handshake messages in TLS
```

Comment:
I don't think this is the right place. Placing it here suggests it's only of interest to people who call `mbedtls_ssl_conf_max_frag_len()` while it actually concerns everyone.

I think the right place would be `ssl_handshake` - though I agree we should add a pointer here, because people who request a small fragment size are more likely to see the server fragment messages than people who don't.